docs/tutorials/gettingstartedexample/gse-tap-basic.md

```diff
@@ -15,20 +15,28 @@ This tutorial will teach you the steps needed to configure a tap acquisition usi
 
-| **Interface Module**   | **Camera**                                                                                                                                                 |
-|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| PXIe-1486 (8 In - 954 Deserializer)         | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
-| PXIe-1486 (4 In 4 Out - 953/954 SerDes)     | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
-| PXIe-1487 (8 I - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
-| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes) | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| **Interface Module**                         | **Camera**                                                                                                                            |
+|----------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| PXIe-1486 (8 In - 954 Deserializer)          | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
+| PXIe-1486 (4 In 4 Out - 953/954 SerDes)      | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
+| PXIe-1487 (8 In - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)  | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1488 (8 In - 9702 Deserializer)         |                                                                                                                                       |
+| PXIe-1488 (4 In 4 Out - 971/9702 SerDes)     |                                                                                                                                       |
+| PXIe-1489 (4 In - 96792A Deserializer)       |                                                                                                                                       |
+| PXIe-1489 (2 In 2 Out - 96793/96792A SerDes) |                                                                                                                                       |
```

Comment:
(Same comment as from the acq doc update)
Why are we listing blank rows in the table? Is it just to convey that this module doesn't have a script that is provided?

If scripts are coming soon, then blank rows seems ok, but if nota nother option, remove the blank rows and change the last sentence above the table to read:

"For modules & cameras not listed in the table below, configuration scripts need to be provided separately."

Reply:
I removed the rows from the table for modules that do not have camera configuration scripts provided with the getting started example.